storage_service/locations/forms.py

```diff
@@ -189,6 +191,7 @@ def __init__(self, *args, **kwargs):
         self.fields['purpose'].widget.disabled_choices = blacklist
         # Associated with all enabled pipelines by default
         self.fields['pipeline'].initial = models.Pipeline.active.values_list('pk', flat=True)
+        self.fields['default'].initial = self.instance.default
```

Comment:
And here is where we pick the current value from the model. `default` is a `@property`, you'll see that later.